test/spec/spectest.cpp

```diff
@@ -612,44 +628,45 @@ void SpecTest::run(std::string_view Proposal, std::string_view UnitName) {
         return;
       }
       case CommandID::AssertTrap: {
-        const auto &Action = Cmd["action"s];
-        const auto &Text = Cmd["text"s].Get<std::string>();
-        const uint64_t LineNumber = Cmd["line"].Get<uint64_t>();
-        TrapInvoke(Action, Text, LineNumber);
+        const simdjson::dom::object Action = Cmd["action"].get_object();
+        const std::string_view Text = Cmd["text"];
+        const uint64_t LineNumber = Cmd["line"];
+        TrapInvoke(Action, std::string(Text), LineNumber);
         return;
       }
       case CommandID::AssertExhaustion: {
         // TODO: Add stack overflow mechanism.
         return;
       }
       case CommandID::AssertMalformed: {
-        const auto &ModType = Cmd["module_type"s].Get<std::string>();
-        if (ModType != "binary") {
+        std::string_view ModType = Cmd["module_type"];
+        std::string Binary = "binary";
```

Comment:
Redundant variables. Use string literals instead.

Reply:
Using string variables in https://github.com/WasmEdge/WasmEdge/pull/2659/commits/b903dff5d269911bd3a841257eba8f95fe440630